frontend/packages/console-plugin-sdk/src/codegen/index.ts

```diff
@@ -1,61 +1,86 @@
 /* eslint-env node */
 
+// eslint-disable-next-line no-restricted-imports
+import * as _ from 'lodash';
 import * as path from 'path';
 import * as readPkg from 'read-pkg';
 
 export type Package = readPkg.NormalizedPackageJson;
 
-export interface PluginPackage extends Package {
+export type PluginPackage = Package & {
   consolePlugin: {
     entry: string;
+    models?: string;
   }
-}
+};
 
-export function isValidPluginPackage(pkg: Package): pkg is PluginPackage {
```

Comment:
As discussed earlier, I've adapted `function` declarations to `const` since the latter form seems to be preferred.